asab/api/doc.py

```diff
@@ -89,9 +89,14 @@ def build_swagger_specs(self):
 			},
 		}
 
+		# Get rid of securitySchemes if there is no authorizationUrl or tokenUrl
+		if not self.AuthorizationUrl or not self.TokenUrl:
+			specs["components"].pop("securitySchemes")
+
 		# Gets all the scopes from config and puts them into scopes
-		for scope in self.Scopes:
-			specs["components"]["securitySchemes"]["oAuth"]["flows"]["authorizationCode"]["scopes"].update({scope: "{} scope.".format(scope.strip().capitalize())})
+		if self.Scopes:
```

Comment:
if I configure scopes and not authorization url, which is stupid, but it might happen, do I get KeyError here?